README.md

```diff
@@ -20,11 +20,19 @@ with a justification and a reference to an issue is required.
 
 ## Package
 
+In the declarative part of both package specifications and bodies all
+declarations of types, subprograms, etc. are separated by a single blank line.
+This includes a single blank line at the beginning and the end of the package.
+The package initialization part after `begin` in both, specification and body,
+should not contain any blank lines.
+
 ### Specification
 
 ```Ada
 package Foo is
+
    <declarations>
+
 end Foo;
 ```
```

Comment:
Maybe `<declarations>` could be split into two to make clear that there is blank line between each declaration:

```Ada
package Foo is

   <declaration>

   <declaration>

end Foo;
```

Reply:
+1